tests/framework/integration/cluster.go

```diff
@@ -503,11 +480,17 @@ func (c *Cluster) waitVersion() {
 
 // isMembersEqual checks whether two members equal except ID field.
 // The given wmembs should always set ID field to empty string.
-func isMembersEqual(membs []client.Member, wmembs []client.Member) bool {
+func isMembersEqual(membs []*pb.Member, wmembs []*pb.Member) bool {
```

Comment:
'getter' that modifies its parameter is evil (I see its not a regression). 
But if you change the code -> maybe its better to afford the copy.

Reply:
I will look into a fix, however I think that code before also mutated original slice. Passing a slice, copies the slice, but not a reference to array underneath. Resting ID in this function is also evil.